tests/test_repro_bug.py

```
@@ -0,0 +1,83 @@
+    async with _bearer_only_client(app, "garbage-token") as ac:
+        resp = await ac.post("/api/decisions/dec-xyz/answer/agent", json={})  # invalid: missing value -> 422
+    print("ANSWER/agent garbage+invalid body ->", resp.status_code, resp.text)
+    assert resp.status_code == 401
+
+
+@pytest.mark.asyncio
+async def test_repro_valid_agent_invalid_body_creates_422(client):
+    """valid token + invalid body must still 422."""
+    app = client._transport.app
+    pid = await _new_project(client)
+    cid, token = await _mint_agent(app, pid, ("decisions_write",))
```

Ruff (0.16.0)

[warning] Unpacked variable `cid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)